src/main.py

```diff
@@ -68,13 +88,21 @@ def __init__(
             huntingLimit: int,
             initialIndividuals: int,
             maximumIndividuals: int
+            calfMortalityRate: float,       # Mortality rate for age < 1
+            youngMortalityRate: float,      # Mortality rate for 1 <= age < 5
+            matureMortalityRate: float,     # Mortality rate for 5 <= age < 12
+            oldMortalityRate: float         # Mortality rate for age >= 12
                  ):
         self.sampleSpace = sampleSpace,                 # S
         self.maxCapacityImpact = maxCapacityImpact,     # c
         self.capacityCurveSlope = capacityCurveSlope,   # a
         self.huntingLimit = huntingLimit,               # l
         self.initialIndividuals = initialIndividuals,   # i_init
         self.maximumIndividuals = maximumIndividuals    # i_max
+        self.calfMortalityRate = calfMortalityRate
+        self.youngMortalityRate = youngMortalityRate
+        self.matureMortalityRate = matureMortalityRate
+        self.oldMortalityRate = oldMortalityRate
```

Comment:
<img width="595" alt="Screenshot 2024-11-07 at 12 32 32" src="https://github.com/user-attachments/assets/9cb01437-7fbf-4c29-b1b5-2690a36eb512">
Mortality rates are not defined by parameters, but by this algorithm from the paper.